build/victoriametrics/build.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2024 Guo-Rong Koh
+
+. ../../lib/build.sh
+
+PROG=victoriametrics
+VER=1.102.1
+PKG=ooce/database/victoriametrics
+SUMMARY="VictoriaMetrics"
+DESC="Fast, cost-effective monitoring solution and time series database."
+
+DATA=var/lib/victoria-metrics-data
+
+set_arch amd64
+set_gover
+
+XFORM_ARGS="
+    -DPREFIX=${PREFIX#/}
+    -DPROG=$PROG
+    -DUSER=$PROG
+    -DGROUP=$PROG
+    -DDATA=$DATA
+    -DVERSION=$VER
+    -DVM=victoria-metrics
+    -DVMAGENT=vmagent
+"
+
+build() {
+    pushd $TMPDIR/$BUILDDIR > /dev/null
+
+    logmsg "Building $PROG"
+    export CGO_ENABLED=0
+    export GOOS=illumos
+
+    logcmd gmake victoria-metrics-pure || logerr "Unable to build victoria-metrics-pure"
```

Comment:
Please can you wrap this line to bring it under 80 characters, and use `$MAKE`?
```suggestion
    logcmd $MAKE victoria-metrics-pure \
        || logerr "Unable to build victoria-metrics-pure"
```